### ISSUE TITLE
crash when parsing string with an emoji

### ISSUE THREAD

**Issue by DetachHead** [sunnyadn/js-toml#2]
```js
const load = require('js-toml').load;
load('asdf = "🔖"')
```
```
Uncaught Xo SyntaxParseError: Syntax error
unexpected character: ->"<- at offset: 7, skipped 4 characters.
```

**Comment by ringohoffman** [comment]
@sunnyadn any chance you can take a look at this?

**Comment by DetachHead** [comment]
@ringohoffman if you're here from pyright, i recommend using my fork [basedpyright](https://docs.basedpyright.com) instead. i switched to a different toml parser library that doesn't have this issue

**Comment by sunnyadn** [comment]
Thanks @DetachHead @ringohoffman for letting me know! I just got back from vacation, but I'll check out the issue soon!

**Comment by sunnyadn** [comment]
Hey @DetachHead @ringohoffman! Just wanted to let you know that I've released version 1.0.1, which includes the fix we discussed.
